.github/workflows/ci.yml

```diff
@@ -59,4 +59,4 @@ jobs:
 
       - name: Test workflow parameters
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test_align_only,docker --outdir ./results
+          nextflow run ${GITHUB_WORKSPACE} -profile test_parameters,docker --outdir ./results
```

Comment:
Don't specially like the name of the profile, `parameters` could be any parametrization of the pipeline?

Reply:
at the moment not yet, it just checks some. 

But i have an open issue to change that into a matrix of all possible combination of parameters as in atac seq :) That is why i changed the name now, but it will need some work i the next PR